bool track_flag
string dock_side
uint8 target_ip
---
string state
